manager/vm/vsock.go

```diff
@@ -1,6 +1,6 @@
 // Copyright (c) Ultraviolet
 // SPDX-License-Identifier: Apache-2.0
-package manager
+package vm
```

Comment:
This probably should also be a part of `qemu` packages since we will potentially use different ways to communicate with Agent. Let's keep it here for the time being, this is just a remark so we don't forget to move it later.